biapol_utilities/label/_match_labels.py

```diff
@@ -42,47 +45,34 @@ def match_labels(label_image_x, label_image_y, method=intersection_over_union_ma
         Image that should serve as a reference for label-matching
     label_image_y : nd-array
         Image the labels of which should be paired with labels from imageA
-    method : str, optional
-        Pairing method to be used. The default is 'iou' (intersection over union).
-        
-        *iou (intersection over union)*: The intersection over union is used to 
-            measure the overlap between labels in both images. The label with
-            highest overlap is chosen as counterpart label
-            
-            iou_streshold: float, optional
-                Threshold value above which two labels are accepted as overlapping.
-                The default value is 0.25
+    method : callable, optional
+        Method to be used for matching the labels. This function is supposed to return a binary matrix with
+        `shape=(n+1, m+1)` corresponding to `match=1`, `no_match=0` of the `n` labels in `label_image_x` and `m` labels in
+        `label_image_y`. The default is thresholded_intersection_over_union_matrix.
+
 
     Returns
     -------
     nd-array
-        Processed version of imageB with labels corresponding to imageA.
+        Processed version of label_image_y with labels corresponding to label_image_x.
     """
     
-    if method == intersection_over_union_matrix:
-        threshold = kwargs.get('iou_threshold', 0.25)
+    method_metric = kwargs.get('metric', intersection_over_union_matrix)
```

Comment:
What speaks agains having three parameters in the function declaration that are called `method_metric`, `method_filter` and `method matching`? We could then also describe these three parameters in the docstring of the function,...

Reply:
Hm...I thought I had already put that in the PR. Maybe I forgot to push it - It should be part of the latest commit (d25db15eaf4afc8774e16ecc22ff3e5b37f28d32)

Reply:
I think these three lines should not be here. If you add these three parameters to the parameter list as any other parameter, this rather complicated stuff is not necessary